src/driver/sqlx_sqlite.rs

```diff
@@ -48,8 +48,8 @@ impl SqlxSqliteConnector {
             .url
             .parse::<SqliteConnectOptions>()
             .map_err(sqlx_error_to_conn_err)?;
-        if options.sqlcipher_key.is_some() {
-            opt = opt.pragma("key", options.sqlcipher_key.clone().unwrap());
+        if let Some(ref sqlcipher_key) = options.sqlcipher_key {
```

Comment:
It really could have been:

```suggestion
        if let Some(sqlcipher_key) = &options.sqlcipher_key {
```

`ref` is really a relic now..

Reply:
https://github.com/SeaQL/sea-orm/pull/1231/commits/cb9cd13ee67bbb111b8ad37e63814e11116598bd